Explorer/Assets/DCL/Interaction/Raycast/Systems/ExecuteRaycastSystem.cs

```diff
@@ -128,6 +142,26 @@ private void BudgetAndExecute(Vector3 scenePosition)
             });
         }
 
+        [Query]
+        [None(typeof(DeleteEntityIntention), typeof(PartitionComponent))]
+        private void GatherSpecialEntitiesRaycastIntents(ref CRDTEntity crdtEntity,
```

Comment:
we can't rely on the absence of `typeof(PartitionComponent)` as partitioning is an asynchronous process (jobified) and finishing for one frame is a particular case that you can't rely on